src/sentry/web/urls.py

```diff
@@ -1025,6 +1027,17 @@
                     react_page_view,
                     name="sentry-organization-replay-rage-clicks",
                 ),
+                # Templates + auth for installing dev toolbar on 3rd party sites
+                re_path(
+                    r"^(?P<organization_slug>[^/]+)/toolbar/iframe/$",
+                    IframeView.as_view(),
+                    name="sentry-organization-toolbar-iframe",
+                ),
+                re_path(
+                    r"^(?P<organization_slug>[^/]+)/toolbar/login-success/$",
+                    LoginSuccessView.as_view(),
+                    name="sentry-organization-toolbar-login-success",
+                ),
```

Comment:
Actually, lets change these again!

Lets do `/toolbar/<org>/<project>/iframe/`  & `/toolbar/<org>/<project>/login-success/`  so we have:
- our own namespace
- both the vars in there (better with own namespace)
- no confusion around whether we need to pair them up like: `/organizations/<org>/toolbar/*/` and `/toolbar/*/`